operations/mimir-mixin/config.libsonnet

```diff
@@ -611,6 +611,10 @@
         enabled: false,
         hpa_name: 'keda-hpa-ruler',
       },
+      gateway: {
+        enabled: false,
+        hpa_name: 'keda-hpa-cortex-gw',
```

Comment:
At grafana labs we also have cortex-gw-internal. I think would be handy if this is a regex matching the internal too. WDYT?

Reply:
Checking.

Reply:
@pracucci I converted the HPA matching to regex, and changed `hpa_name` into `keda-hpa-cortex-gw.*`. From my testing, this causes `keda-hpa-cortex-gw-internal` to also be included in the auto-scaling panels. Do you think this is a good solution?"